proto/buf/alpha/registry/v1alpha1/organization.proto

```diff
@@ -232,8 +232,6 @@ message GetOrganizationSettingsRequest {
 
 message GetOrganizationSettingsResponse {
   RepositoryRole repository_base_role = 1;
-  PluginRole plugin_base_role = 2 [deprecated = true];
```

Comment:
shouldn't these be reserved?

Reply:
The UI should be the sole consumer of these, and the underlying code has already been removed (treating this as unspecified).

We could reserve it, but given these are "internal" API contracts I'm not sure how much value there is?

Reply:
Sure, I'll reserve the definitions in this PR. 

Do you want me to backfill reserved for definitions from prior PR? E.g., https://github.com/bufbuild/buf/pull/2278/files?file-filters%5B%5D=.proto&show-deleted-files=true&show-viewed-files=true#r1258876594

Reply:
optimally if its quick